docs/userguide/dev/codingguidelines.md

```diff
@@ -445,6 +445,24 @@ type `UBool` must be used in public APIs and in the definition of any persistent
 data structures. `UBool` is guaranteed to be one byte in size and signed; bool is
 not.
 
+Traditionally, ICU4C has defined its own FALSE=0 / TRUE=1 macros for use with UBool.
+Starting with ICU 68 (fall 2020), we no longer define these in public header files
+(unless U_DEFINE_FALSE_AND_TRUE=1),
```

Comment:
super nit: could use back-tick (\`) for macro names (similar to the above `UBool`) for syntax highlighting in the rendered markdown output.

Reply:
done